main.tf

```diff
@@ -13,10 +13,17 @@ data "aws_vpc" "default" {
 
 data "aws_availability_zones" "available" {}
 
+resource "null_resource" "depends_on" {
```

Comment:
If we return `igw_id` (see #11), this resource may be not required.

Reply:
Returned